pkg/data/management/role_data.go

```diff
@@ -403,9 +398,7 @@ func addUserRules(role *roleBuilder) *roleBuilder {
 		addRule().apiGroups("management.cattle.io").resources("clusters").verbs("create").
 		addRule().apiGroups("management.cattle.io").resources("nodedrivers").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("kontainerdrivers").verbs("get", "list", "watch").
-		addRule().apiGroups("management.cattle.io").resources("nodetemplates").verbs("create").
 		addRule().apiGroups("management.cattle.io").resources("fleetworkspaces").verbs("create").
-		addRule().apiGroups("management.cattle.io").resources("multiclusterapps", "clustertemplaterevisions").verbs("create").
```

Comment:
Also remove `rkek8ssystemimages`, `rkek8sserviceoptions` and `rkeaddons`

Reply:
done